src/legacy/ui/public/courier/fetch/fetch_soon.js

```diff
@@ -30,13 +30,12 @@ import { FetchNowProvider } from './fetch_now';
  * @param Promise
```

Comment:
mega-nit: should probably either delete the JSDoc syntax here, or add `@param config`